src/main/java/Daniela_Chihai/com/Endava/SoA5/Java_References/Arrays_Ex1.java

```diff
@@ -0,0 +1,35 @@
+package Daniela_Chihai.com.Endava.SoA5.Java_References;
+
+public class Arrays_Ex1 {
+    public static int numbersMultiplication(int[] a) {      //banana is the method; int[] a{3,8,9} - array of parameters
+        int product = 1; // initializing the first value of the product
+        int i = 0;   // initialization of i with 0
+        while (i < a.length) {  // while i { from 0....to length of a}
+            product = product * a[i];  // multiplying each element of the array with the next one
+            i++;   // it increments the array with the next value
+        }
+        return product;
+    }
+    // This method calculates the product of multiplication of each element from the array of numbers between them.
+
+    public static int getIndex(int[] a, int num) { // "int num" is an external variable. We should declare it in the main method when running program
```

Comment:
The same here. Please do this wherever you use this "int[] a"

Reply:
Done